sift-for-woocommerce.php

```diff
@@ -45,7 +45,7 @@ static function () {
 	add_action(
 		'admin_notices',
 		static function () {
-			$message      = __( 'It seems like <strong>Sift for WooCommerce</strong> is corrupted. Please reinstall!', 'sift-for-woocommerce' );
+			$message      = __( 'It seems like <strong>Sift Decisions</strong> is corrupted. Please reinstall!', 'sift-for-woocommerce' );
```

Comment:
Probably should change this back to Sift For WooCommerce?

Reply:
You're right. It was an oversight.